modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java

```diff
@@ -689,6 +689,32 @@ private ObjectDefinition _createFDSViewObjectDefinition(
 			fdsViewObjectDefinition.getObjectDefinitionId(),
 			labelObjectField.getObjectFieldId());
 
+		if (FeatureFlagManagerUtil.isEnabled("LPD-10735")) {
+			ObjectField defaultViewObjectField =
```

Comment:
Don't know if it is worth all this boilerplate to add a field under a FF. 
An option could be to customize the [_addLocalizedCustomObjectField](https://github.com/liferay-frontend/liferay-portal/blob/5f5adf9cd182d06f693d01128a59600f259f47f9/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/portlet/FDSViewsPortlet.java#L150) and allow creating localized/non-localized custom object fields.

Reply:
Forgot to say, anyways, I'm ok with the code even if it's not optimal. In the end we'll end up removing it soon when migrating the data model

Reply:
If we are not going to break anything, my bet would be adding the field directly, without FF.